ui/src/components/SampleView/DrawGridPlugin.js

```diff
@@ -225,6 +225,9 @@ export default class DrawGridPlugin {
    * @param {float} y - bottom y coordinate of grid, (mouse y position)
    */
   update(canvas, x, y) {
+    this.gridData.screenCoord[0] = this.currentTopLeftX;
```

Comment:
Making sure that the grid coordinates are not being updated on accidental clicks (after being drawn but not saved)